alerter.c

```diff
@@ -26,6 +39,7 @@ void alertInCelcius(float farenheit) {
 int main() {
     alertInCelcius(400.5);

```

Comment:
Boundary values need to be checked. Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold. If threshold is 392 F (i.e. 200 c) then consider values <392, =392 and >392

Reply:
Done